client/src/pages/Register.js

```diff
@@ -38,26 +90,57 @@ const Register = () => {
 				headers: { "Content-Type": "application/json" }
 			});
 
-			// Configure later for snackbars
+			// Redirect them to messenger page
+			if (res.status === 201) {
+				history.push("/messenger");
+			}
+
 			// In the event we get an error
 			const data = await res.json();
 			if (data.errors) {
-				console.log(data);
+				if (data.errors.email) {
+					setSnackbar({
+						open: true,
+						severity: "error",
+						message: data.errors.email
+					});
+				}
 			}
-
-			// Redirect them to messenger page
-			// history.js stuff
 		} catch (err) {
 			console.log(err);
 		}
 	};
-	// Responsive breakpoints
-	const isMobile = useMediaQuery({
-		query: "(max-device-width: 767px)"
-	});
-	const isTabletDesktop = useMediaQuery({
-		query: "(min-device-width: 768px)"
-	});
+
+	// Pressing enter submits our form
+	const handleKeyPress = (e) => {
+		if (e.which === 13) {
+			handleSubmit();
+		}
+	};
+
+	// Closes our snackbar
+	const handleClose = (event, reason) => {
+		if (reason === "clickaway") {
+			return;
+		}
+
+		setSnackbar({ ...snackbar, open: false });
+	};
+
+	// Styles
+	const styles = {
+		loginButton: {
```

Comment:
not that there's anything wrong with this but you may find it beneficial to be leverage Material UI's makeStyles (https://material-ui.com/styles/basics/) so you can follow the convention of using classNames instead of overwriting its `styles` attribute

Reply:
I have decided to migrate these styles to my stylesheet but I'll try looking into this more in-depth once I start developing the front end part of the messaging application!